node/pkg/winupgrade/upgrade_test.go

```diff
@@ -40,9 +38,6 @@ const (
 var _ = DescribeTable("verifyImagesShareRegistryPath",
 	func(upgradeImage string, nodeImage string, noError bool) {
 		err := verifyImagesSharePathPrefix(upgradeImage, nodeImage)
-		if err != nil {
-			fmt.Println(err)
-		}
 		Expect(err == nil).To(Equal(noError))
```

Comment:
Not your change, but this is a bit bizarre - should be `Expect(err).NotTo(HaveOccurred())`